pkg/work/webhook/start.go

```diff
@@ -44,32 +44,34 @@ func (c *Options) RunWebhookServer() error {
 		}),
 	})
 
+	logger := klog.LoggerWithName(klog.FromContext(context.Background()), "work webhook")
```

Comment:
Consider to move the logger init lines to the beginning of the `RunWebhookServer` method. so it is easy to understand the err check in line 49 is for the `ctrl.NewManager`

Reply:
updated